doc/src/Howto_moltemplate.rst

```diff
@@ -64,28 +69,27 @@ The resulting file (``formamide.lt``) follows:
      }
    }
 
-You don't have to specify the charge in this example because they will
-be assigned according to the atom type. Analogously, only a
-"Data Bond List" section is needed as the atom type will determine the
-bond type. The other bonded interactions (e.g. angles,
-dihedrals, and impropers) will be automatically generated by
+You don't have to specify the charge in this example because the OPLSAA
+force-field assigns charge according to the atom type.  (This is not true
+when using other force fields.)  A "Data Bond List" section is needed as
+the atom type will determine the bond type. The other bonded interactions
+(e.g. angles, dihedrals, and impropers) will be automatically generated by
 Moltemplate.
 
 If the simulation is non-neutral, or Moltemplate complains that you have
-missing bond, angle, or dihedral types, this means at least one of your
-atom types is incorrect.
+missing bond, angle, or dihedral types, this probably means at least one
```

Comment:
Suggestion: `means at least..` --> `means that at least..`; `(or perhaps there is..` --> `(or that perhaps there is..`